src/GraphODataTemplateWriter/GraphODataTemplateWriter.csproj

```diff
@@ -71,15 +67,13 @@
     <Compile Include="CodeHelpers\Python\CodeWriterPython.cs" />
```

Comment:
any specific reason we've kept python?

Reply:
Darrel said that we can keep it if we can take out anything useful when we revisit Python. I didn't get a chance to judge whether there is anything useful, so I kept it for now.

Reply:
It is still useful at this time to generate models for Python. The request builders aren't useful. We've provided them to an internal team once more on as-is basis. Let's revisit when we finalize Python SDK models.